app/index.tsx

```diff
@@ -135,6 +140,8 @@ export default class Root extends React.Component<{}, IState> {
 			return;
 		}
 
+		await getInitialNotification();
```

Comment:
On Android, wouldn't we fetch the same notification twice, since we do `initializePushNotifications` and `getInitialNotification`?

Reply:
I thought the same thing would happen, but then I saw that this method only picks up pushes sent via notifiee